Scripts/Linux/InstallFromSource.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+# To test on a Debian-based Linux machine:
+#
+# ./InstallFromSource.sh
+# (outputs Git, GCM, and Scalar versions)
+#
+# Small clone should work:
+# scalar clone https://dev.azure.com/gvfs/ci/_git/ForTests
+#
+# Big clone for authenticated repo should work:
+# scalar clone <url>
+#
+# Note: if you are using a headless Linux machine, then GCM Core will
+# not be able to store your credentials. Instead, create a PAT and
+# use that for password (multiple times). This will be fixed soon.
+
+# Nuke previous data
+rm -rf git-vfs*.deb gcmcore-linux*.deb packages*.deb
+
+echo "Installing dotnet SDK 3.1"
+
+sudo apt-get update -qq
+sudo apt-get install -y apt-transport-https && sudo apt-get update -qq
+
+sudo apt-get install -y wget
+wget https://packages.microsoft.com/config/ubuntu/$(lsb_release -rs)/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
```

Comment:
On my debian system, `lsb_release -rs` returns "unstable", so this `wget` gets a 404. You might also need to make sure `lsb-release` is installed (it's priority "optional" in debian).

Reply:
I will make sure any errors in this section report a message saying "Go <here> to figure out dotnet for yourself!"

Reply:
Good points -- maybe something like:
```shell
sudo apt-get install -y lsb-release wget
LSB_VERSION=$(lsb_release -rs | awk '/^[0-9]+\.[0-9]+$/ { print }')
if [ -z "$LSB_VERSION" ]; then LSB_VERSION="20.04"; fi
wget https://packages.microsoft.com/config/ubuntu/"$LSB_VERSION"/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
```